docs/Site-Reliability-Guardian/example-pipelines/Github/readme.md

```diff
@@ -1,29 +1,37 @@
 # Github integration
 
-To integrate the `dta` CLI with GitHub for the quality gates use case you can use the action `dta-docker` that is in this folder. This action takes the docker image for the CLI and executes the command `/dta srg evaluate` to trigger a new quality gate evaluation. (Note: You need to first manually configure the Dynatrace Workflow and SRG evaluation as described in the main docs).
+To integrate the `dta` CLI with GitHub for the quality gates use case you can use the [workflow pipeline in this folder](./github.yml) (Note: You need to first manually configure the Dynatrace Workflow and SRG evaluation as described in the main docs).
 
-You can consume the Github action in a workflow like the following example:
+You can use the Github workflow by running the evaluation job:
 
 ```
-
-       - id: build-release
-        name: Run CLI with SRG use case
-        uses: ./.github/actions/dta-docker #update this reference to where you are hosting the dta-docker action
-        with:
-          dt-url: ${{ inputs.DYNATRACE_URL_GEN3 }}
-          dt-account-urn: ${{ secrets.ACCOUNT_URN }}
-          dt-client-id: ${{ secrets.DYNATRACE_CLIENT_ID }}
-          dt-client-secret: ${{ secrets.DYNATRACE_SECRET }}
-          dt-sso-url: ${{ secrets.DYNATRACE_SSO_URL }}
-          srg-timespan: "5" #this would grab the last 5 mins
-          dt-service-name: "demo-cli-service"
-          dt-stage-name: "dev"
+  evaluation:
+    needs: test
+    runs-on: ubuntu-latest
+    container:
+      image: "dynatraceace/dt-automation-cli:latest"
+    steps:
+      - name: "Run SRG evaluation"
+        run: |
+          dta srg evaluate
+        env:
+          LOG_LEVEL: "verbose"
+          DYNATRACE_URL_GEN3: ${{ secrets.DYNATRACE_URL_GEN3 }}
+          ACCOUNT_URN: ${{ secrets.ACCOUNT_URN }}
+          DYNATRACE_CLIENT_ID: ${{ secrets.DYNATRACE_CLIENT_ID }}
+          DYNATRACE_SECRET: ${{ secrets.DYNATRACE_SECRET }}
+          DYNATRACE_SSO_URL: ${{ secrets.DYNATRACE_SSO_URL }}
+          SRG_EVALUATION_STOP_ON_FAILURE: "true"
+          SRG_EVALUATION_SERVICE: "demo"
+          SRG_EVALUATION_STAGE: "dev"
+          SRG_EVALUATION_START_TIME: "${{ needs.test.outputs.startTime }}"
+          SRG_EVALUATION_END_TIME: "${{ needs.test.outputs.endTime }}"
```

Comment:
we can also add app version and buildId variables in this example

Reply:
good idea, let me fix the other PR first and then merge it here

Reply:
ok agreed